apps/frontend/src/libs/modules/notifier/notifier.module.ts

```diff
@@ -0,0 +1,21 @@
+import { toast } from "react-toastify";
```

Comment:
file name and folder name should be the same as module name. `modules/toast-notifier/toast-notifier.ts`